### ISSUE TITLE
AttributeError: '_RSAPrivateKey' object has no attribute 'signer'

### ISSUE THREAD

**Issue by nagappa2011** [mosen/SCEPy#4]
I was just going through this implementation and it is really helpful.. 
I am getting this error when i try to generate pkcs7 request for certificate download. Since I  need to use cryptography 38.0.3 for my other dependencies , Need to fix this issue in that version itself. Any Suggestions will be helpful . 
Thanks in Advance..
Best Regards

**Comment by mosen** [comment]
Sorry i was not aware that a newer cryptography broke the package. I will have a look as soon as i can

**Comment by nchinnannava-JNPR** [comment]
Sure.. Thank a lot Mosen.. I think Signer and also signer_cert.public_key().verifier()  both the places need changes... And One more thing what i observed is der_req = req.get_decrypted_envelope_data().. Not able to download CSR from der_req.. it complains about 6 extra bytes..

**Comment by toddjcrane** [comment]
Apparently it was pulled from pyca/cryptography a little over a year ago after being depreciated for 4 years.    https://github.com/pyca/cryptography/pull/6639

**Comment by mosen** [comment]
yeah apologies i have not taken this project any further since the breakage. I can look at it but have very little spare time at the moment.
